### PR TITLE
[update][flink cdc] Code reduction

### PR DESCRIPTION
**Pull request by rookiegao**
The scala code conflicts with the current Java code, Only Java code is reserved temporarily

### CODE REVIEW

File: Bigdata-flink-example/Flink-cdc-example/pom.xml [hunk 1]
```diff
@@ -147,7 +147,7 @@
             <groupId>org.scala-lang</groupId>
             <artifactId>scala-library</artifactId>
             <version>2.12.12</version>
-            <scope>compile</scope>
```
Comment (xingjieshitu) [inline Bigdata-flink-example/Flink-cdc-example/pom.xml:150]: What is the impact of this change?

**Review by xingjieshitu — COMMENTED** [review]
What exactly is the conflict?

File: Bigdata-flink-example/Flink-cdc-example/pom.xml (hunk 1 above)
Comment (rookiegao) [reply Bigdata-flink-example/Flink-cdc-example/pom.xml:150]: We don't need to break into Scala dependency

**Comment by rookiegao** [comment]
> 

As shown in the title

**Review by xingjieshitu — APPROVED** [review]

**Pull request merged**